nativelib/src/main/resources/gc/immix/headers/BlockHeader.h

```diff
@@ -20,85 +20,97 @@ typedef struct {
         int16_t first;
         int32_t nextBlock;
     } header;
-    LineHeader lineHeaders[LINE_COUNT];
 } BlockHeader;
 
-static inline bool Block_IsRecyclable(BlockHeader *blockHeader) {
+static inline bool BlockHeader_IsRecyclable(BlockHeader *blockHeader) {
```

Comment:
Block metadata is not stored in the header (initial part) of the block. So it's best to rename it to be `BlockMetadata` (or `BlockMeta` to shorten it a bit) instead of `BlockHeader`.

Reply:
Similarly accessing a bytemap, gives you `ObjectMeta`.